ethers-signers/src/ledger/mod.rs

```diff
@@ -25,6 +29,16 @@ impl Signer for LedgerEthereum {
         self.sign_tx(message).await
     }
 
+    async fn sign_typed_data<T: Eip712 + Send + Sync>(
+        &self,
+        payload: T,
+        domain: Option<EIP712Domain>,
+    ) -> Result<Signature, Self::Error> {
+        let hash = payload.encode_eip712(domain)?;
+
+        Ok(self.sign_hash(hash.into(), false))
```

Comment:
It seems that `sign_hash` is not available on `LedgerEthereum` which you are implementing for here. Should this be eventually `sign_message?`